contrib/dockerfiles/defi.dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG TARGET=x86_64-pc-linux-gnu
+
+FROM --platform=linux/amd64 ubuntu:latest as defi
+ARG TARGET
+ARG PACKAGE
+ENV PATH=/app/bin:$PATH
+LABEL org.defichain.name="defichain"
+LABEL org.defichain.arch=${TARGET}
+
+WORKDIR /app
+COPY ${PACKAGE} ./
+RUN tar -xvzf ${PACKAGE} --strip-components 1
```

Comment:
This puts both on the image, the tar as well extracted versions, perhaps add another build state to have it clean?

Reply:
Fixed in PR #2721